tensorflow/core/kernels/list_kernels.cc

```diff
@@ -33,6 +31,7 @@ limitations under the License.
 #include "tensorflow/core/lib/core/coding.h"
 #include "tensorflow/core/lib/core/errors.h"
 #include "tensorflow/core/util/util.h"
+#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor" #include "tensorflow/core/framework/op_kernel.h"
```

Comment:
Why has it been modified?

Reply:
Mistake? Looks like you joined the two lines together. :)

Reply:
It was modified by the clang-format. But no change here, so I reverted.

---
In reply to: [524980697](https://github.com/microsoft/tensorflow-directml/pull/95#discussion_r524980697) [](ancestors = 524980697)